plugins/nextcloud-rule-exclusions-before.conf

```diff
@@ -465,6 +465,17 @@ SecRule REQUEST_FILENAME "@rx /(?:remote|index|public)\.php/" \
     ver:'nextcloud-rule-exclusions-plugin/1.1.0',\
     setvar:'tx.allowed_methods=%{tx.allowed_methods} PUT PATCH CHECKOUT COPY DELETE LOCK MERGE MKACTIVITY MKCOL MOVE PROPFIND PROPPATCH SEARCH UNLOCK REPORT TRACE jsonp'"
 
+# Allow CalDav/CarDav clients to scan for CalDav CarDav path
+SecRule REQUEST_FILENAME "@streq /" \
```

Comment:
Is it really going to `/` ? Isn't there any way to make this exclusion more tight?

Reply:
Well most dav clients make a request to /, then Nextcloud will redirect them to ``/remote.php/dav/whatever``. This only happens when your configuring a new client, existing clients will go to ``/remote.php/dav/whatever``.